Makefile

```diff
@@ -13,11 +13,19 @@ GOBJS    = main-gui.o gui.o unixtime.o gpx-read.o correlate.o exif-gps.o latlong
 CFLAGS   = -Wall -O2
 CFLAGSINC := $(shell $(PKG_CONFIG) --cflags libxml-2.0 exiv2)
 # Add the gtk+ flags only when building the GUI
+ifeq (${GTK},3)
```

Comment:
This could be done without the GNU-specific ifeq by setting GTK=2 above and using it as gtk+-$(GTK).0 below.

Reply:
Updated.